appcues/src/main/java/com/appcues/debugger/DebuggerRecentEventsManager.kt

```diff
@@ -229,26 +233,29 @@ private fun Map<String, Any>.filterOutScreenProperties(eventType: EventType): Ma
     } else this
 }
 
-private fun Map<String, Any>.filterOutAutoProperties(): Map<String, Any> {
-    return filterNot { it.key == AutoPropertyDecorator.IDENTITY_PROPERTY }
-}
-
-private fun Map<String, Any>.filterOutMetricsProperties(): Map<String, Any> {
-    return filterNot { it.key == SdkMetrics.METRICS_PROPERTY }
+// filter out all internal properties so it doesnt show in "properties" section
+private fun Map<String, Any>.filterOutInternalProperties(): Map<String, Any> {
```

Comment:
got detekt error (too many functions) in this file, since we are only using these filterOut in a single place I think its ok to consolidate all of those into one method.